.pre-commit-config.yaml

```diff
@@ -144,7 +144,7 @@ repos:
     # The `refs/tags/<tag>:refs/tags/<tag>` is needed for black's
     # `required-version` option to work.  See
     # https://github.com/psf/black/issues/2493#issuecomment-1081987650
-    rev: "refs/tags/22.3.0:refs/tags/22.3.0"
+    rev: "22.3.0"
```

Comment:
Don't remove `refs/tags/22.3.0:refs/tags/` in front the version number.  See https://github.com/andthum/hpc_submit_scripts/pull/31#issuecomment-1135534099